scintilla/boostregex/BoostRegExSearch.cxx

```diff
@@ -251,6 +251,17 @@ RegexSearchBase *CreateRegexSearch(CharClassify* /* charClassTable */)
 #endif
 }
 
+void propagateErrorMessage(char const *msg)

```

Comment:
Maybe, just use global string buffer instead of this cumbersome method for message propagation?

Reply:
There is a global string buffer? I was not aware of this. But if it's easy, I'm open to it. Where is the API?

Reply:
@Uhf7 May I push to your PR? You can revert my commit later if you don't like it.

Reply:
Here is the idea: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/3d08ed23302ec544aac5c015c900e980c2ffb90d
You can get this commit to your PR if you like it (git cherry-pick, git apply diff, etc).

Reply:
Ah, now I get it. You want to export a new function from SciLexer.dll. Seems to be the better way here, technically. Thank you. I look into it.